webapp/src/components/call_widget/component.tsx

```diff
@@ -1064,18 +1033,37 @@ export default class CallWidget extends React.PureComponent<Props, State> {
                 >
                     <button
                         className='style--none'
-                        style={{background: device.deviceId === currentDevice?.deviceId ? 'rgba(28, 88, 217, 0.12)' : ''}}
+                        style={{
+                            background: device.deviceId === currentDevice?.deviceId ? 'rgba(28, 88, 217, 0.08)' : '',
```

Comment:
I hate to say, but should this be themed?

Reply:
I think that's already themed. The background is just an overlay when the device is selected. Unless you mean something else?

Reply:
Nope, just wanted to make sure that hardcoded value was intentional and not a missed theming opportunity. Thanks!